webapp/src/components/post_type_zoom/post_type_zoom.jsx

```diff
@@ -177,7 +177,7 @@ export default class PostTypeZoom extends React.PureComponent {
                         className='btn btn-lg btn-primary'
                         style={style.button}
                         rel='noopener noreferrer'
-                        onClick={() => this.props.actions.startMeeting(this.props.currentChannelId, '', true, props.meeting_topic)}
+                        onClick={() => this.props.actions.startMeeting(post.channel_id, post.root_id, true, props.meeting_topic)}
```

Comment:
Not sure if this applies here, but whenever we reference `post.root_id`, we usually have a `|| post.id` to say "and also handle this if the post is itself a root post"

```suggestion
                        onClick={() => this.props.actions.startMeeting(post.channel_id, post.root_id || post.id, true, props.meeting_topic)}
```

Reply:
Just to make sure, there is never any benefit to using `post.id` here

Reply:
@mickmister Yes, it won't be useful